andela/ah-backend-sparta
# #164798201 Email verification after sign up

**What does this PR do?**

- Enables a user, on signing up, to receive an email with a verification link

**Description of Task to be completed?**

Users should receive an email containing an activation link when they successfully signup.
Users' accounts should be activated when they click on the activation link they received in their email

**How should this be manually tested?**

- Fetch this branch `git fetch origin ft-signup-email-verification-164798204`
- Checkout to the branch `git checkout ft-signup-email-verification-16479820`
- add the necessary environment variables to your .env file
- Run `python manage.py migrate`
- Run `python manage.py runserver`
- Sign up at POST /api/users/register/
- Try signing in using POST /api/users/login/
- If it fails, go to your email and click on the link to verify
- Try signing in again

What are the relevant pivotal tracker stories?

[#164798201](https://www.pivotaltracker.com/n/projects/2320796/stories/164798201)

[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/456b792a1e3e866a0baf)
